os/fs/procfs/fs_procfslogsave.c

```diff
@@ -122,7 +122,8 @@ static int logsave_open(FAR struct file *filep, FAR const char *relpath, int ofl
 	FAR struct logsave_file_s *attr = (struct logsave_file_s *)kmm_zalloc(sizeof(struct logsave_file_s));
 	fvdbg("Open '%s'\n", relpath);
 	filep->f_priv = (FAR void *)attr;
-	log_dump_read_init();
+	if(log_dump_read_init() != OK)
```

Comment:
cureently log_dump_read_init sends ok for all the time. you should check error condition of log_dump_read_init.